lib/src/operations/cardano_get_serial_operation.dart

```diff
@@ -0,0 +1,31 @@
+import 'dart:typed_data';
+
+import 'package:buffer/buffer.dart';
+import 'package:ledger_cardano/src/operations/cardano_ledger_operation.dart';
+import 'package:ledger_cardano/src/operations/complex_ledger_operations.dart';
+import 'package:ledger_cardano/src/operations/ledger_operations.dart';
+import 'package:ledger_cardano/src/utils/hex_utils.dart';
+import 'package:ledger_cardano/src/utils/utilities.dart';
+
+class CardanoGetSerialOperation extends ComplexLedgerOperation<String> {
+  CardanoGetSerialOperation() : super();
+
+  @override
+  Future<String> invoke(LedgerSendFct send) async {
+    final Uint8List data = useBinaryWriter((ByteDataWriter writer) {
+      writer.writeUint8(0x00);
+      return writer.toBytes();
+    });
```

Comment:
That's exactly how I'd expect `useBinaryWriter` to be used. Please keep on doing it this way.

Reply:
👍